src/modules/Builder/features/Activities/ActivitiesHeader/ActivitiesHeader.test.tsx

```diff
@@ -0,0 +1,30 @@
+import { render, fireEvent, screen } from '@testing-library/react';
+
+import { ActivitiesHeader } from '.';
+
+describe('ActivitiesHeader - Interaction Tests', () => {
+  test('calls onAddActivity when the Add Activity button is clicked', () => {
+    const mockOnAddActivity = jest.fn();
+
+    render(
+      <ActivitiesHeader headerProps={{ onAddActivity: mockOnAddActivity }}>
+        <></>
+      </ActivitiesHeader>,
+    );
+
+    fireEvent.click(screen.getByTestId('builder-activities-add-activity'));
+    expect(mockOnAddActivity).toHaveBeenCalled();
+  });
+
+  test('opens and closes the performance task menu correctly', () => {
+    render(
+      <ActivitiesHeader>
+        <></>
+      </ActivitiesHeader>,
+    );
+
+    const addPerfTaskButton = screen.getByTestId('builder-activities-add-perf-task');
+    fireEvent.click(addPerfTaskButton);
+    expect(screen.getByText('A/B Trails iPad')).toBeInTheDocument();
```

Comment:
Could you pls add all performance task names?
![image](https://github.com/ChildMindInstitute/mindlogger-admin/assets/116339462/33858527-166f-4967-b1da-3288078f9782)
And also make this menu close, because test description has this point.
"...and closes the performance task menu coorectly"

Reply:
fixed